src/sentry/testutils/pytest/relay.py

```diff
@@ -81,8 +80,8 @@ def relay_server_setup(live_server, tmpdir_factory):
     template_vars = {
         "SENTRY_HOST": f"http://host.docker.internal:{port}/",
         "RELAY_PORT": relay_port,
-        "KAFKA_HOST": "kafka-kafka-1" if use_new_dev_services else "sentry_kafka",
-        "REDIS_HOST": "redis-redis-1" if use_new_dev_services else "sentry_redis",
+        "KAFKA_HOST": "kafka",
```

Comment:
turns out we don't need to use container name here, lookup also works with service name and kafka/redis are cleaner names to use here